Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,9 +4,62 @@ namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
-        }
+        public static void Main()
+            {
+                PersonDetails();
+            }
+
+        public static void PersonDetails()
+            {
+            string firstName, lastName;
+            float age, weight, height;
```

Comment:
You should declare your variables as late as possible. This will spare the reader the time and resources to memorize your variable types throughout the code. You can declare it and initialize it at the line of first usage.

Also note:
If you use the out keyword you can simplify things:
Instead of writing
`float returnValue;`
`float.TryParse(s, out returnValue);`
you can simply write
`float.TryParse(s, out float returnValue);`

You should also use weight or height directly instead of caching it into returnValue only to pass it to the other variable.
Also age should probably be of type int!